cmd/goacme/main.go

```diff
@@ -130,6 +131,15 @@ type command struct {
 	// 'goacme help <command>' output.
 	// The template context is longTemplateData.
 	Long string
+
+	// ConfigDir is the configuration directory which is used
+	ConfigDir string
+
+	// DefaultDisco is the default CA directory endpoint.
+	DefaultDisco string
+
+	// AccountFile is the default user config file name.
+	AccountFile string
```

Comment:
This seems like a substantial change. Not sure these belong to the `command` type.
I need to think about it.